packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -111,6 +111,32 @@ const TEMPLATE_SOURCE_NPM = 'npm'
 const TEMPLATE_SOURCE_BUNDLE = 'bundle'
 const DEFAULT_TEMPLATE_VERSION = 'latest'
 
+const EXTENSION_QUESTIONS = [
+    {
+        name: 'project.generationType',
+        message: 'Do you want to generate a project using extensions or using a template?',
+        type: 'list',
+        choices: [
+            {name: 'Generate a project using extensions', value: 'extensions'},
+            {name: 'Generate a project using a template', value: 'template'}
+        ]
+    },
```

Comment:
This comment might be a little meta, but is this question really categorized as an "extension" question? I feel like the later 2 are, but the first is more like the question that opts you into the extension question flow.

Also, I suggest that we try keeping on brand and call them at a minimum app extensions (maybe when referencing in code as variables) and Application Extensibility (note the capitalization) when using it in static strings.